pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/awsconnectionbasic/_http.log

```diff
@@ -93,8 +94,7 @@ X-Xss-Protection: 0
 {
   "aws": {
     "accessRole": {
-      "iamRoleId": "arn:aws:iam::199999999999:role/basicrole",
-      "identity": "048077221682493034546"
```

Comment:
What happened to this field?

Reply:
This PR based on a pending PR. #3042 (implements the update field mask in MockGCP)
I'll rebase once that other one merges. This should go away